data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"Is this it? Is the game over now that humanity's at peace again?"`
```

Comment:
I edited my approval to also approve of lumbar527's suggestion, and offer a few other options. The following one is, I think, the best, but I don't know what EjoThims and Saugia have in mind:
```suggestion
				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been completed yet."`
```

Reply:
I think reverting the change to "some parts" back to "that part" should be switched, it puts us back to possibly implying that "the rest of the galaxy" is a single whole unit that isn't completed at all, instead of specifically addressing "the rest of the galaxy" as distinct parts that each may or may not be completed.

But I don't have an issue with using "completed" instead of "implemented" here. 👍

Reply:
Changing that to "some parts" is fine with me.

Reply:
I'm a fan of this approach.

Reply:
Which one? The original one in the comment, or the following one in reply to it?

Reply:
I'm assuming the one in reply to it, because if it was the original one Amazinite probably would have responded to the whole PR and not this thread.

Reply:
Ziproot's suggestion. It maintains the idea of the original choice while being more accurate to how the game is today. Sure, there's stuff implemented in the rest of the galaxy now--when this choice was added, all that existed was human space, Hai space, and the Korath exile systems--but it's not fully implemented in the sense that the regions that do exist are unfinished.

Reply:
This suggestion (Ziproot's) is great!